### PR TITLE
Cams 47 remove chapter 11 code

### PR DESCRIPTION
**Pull request by fmaddenflx**
# Purpose

Removed Chapter 11 from front and back end.

# Major Changes

UI no longer has a "cases" link in primary navigation.  All related code has been removed and tests updated.

# Testing/Validation

New tests written and/or updated.

### CODE REVIEW

File: user-interface/src/App.tsx [hunk 1]
```diff
@@ -2,13 +2,12 @@ import './App.scss';
...
 import NotFound from './components/NotFound';
-import { CaseList } from './components/CaseList';
 import { CaseAssignment } from './components/CaseAssignment';
 import { HeaderNavBar } from './components/HeaderNavBar';
 import { Provider } from 'react-redux';
 import { store } from './store/store';
-import { AppInsightsErrorBoundary } from '@microsoft/applicationinsights-react-js';
-import { reactPlugin } from './ApplicationInsightsService';
+//import { AppInsightsErrorBoundary } from '@microsoft/applicationinsights-react-js';
+//import { reactPlugin } from './ApplicationInsightsService';
```
Comment (fmaddenflx) [inline user-interface/src/App.tsx:10]: We need to re-enable App-Insights here

File: user-interface/src/App.tsx [hunk 2]
```diff
@@ -33,14 +33,26 @@ function App() {
           <div className="body">
             <Routes>
               <Route path="/" element={<Home />}></Route>
```
Comment (fmaddenflx) [inline user-interface/src/App.tsx:35]: We need to re-enable App-Insights here

File: user-interface/src/models/attorneys-api.mock.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export default class MockAttorneysApi {
+  public static async getAttorneys() {
+    console.log('mock is called');
+    return Promise.resolve([]);
+  }
+}
```
Comment (fmaddenflx) [inline user-interface/src/models/attorneys-api.mock.ts:6]: Was this file even being used?  I thought we abandoned it.

Comment (fmaddenflx) [reply user-interface/src/models/attorneys-api.mock.ts:6]: Never mind.  We are using it.

**Review by amorrow-flexion — APPROVED** [review]

**Review by jamesobrooks — APPROVED** [review]

**Pull request merged**